### PR TITLE
setquota, cvfsck: add page

### LINKED ISSUE #8512: page request: cvfsck
### Command description

Check and Recover a Xsan Volume.

The cvfsck program can check and repair Xsan file system metadata corruption due to a system crash, bad disk or other catastrophic failure. This program also has the ability to list all of the existing files and their pertinent statistics, such as inode number, size, file type and location in the volume.

### Command details

_No response_

### OS

- [ ] Android
- [X] Linux
- [ ] macOS
- [ ] sunOS
- [ ] Windows

### PR DESCRIPTION
**Pull request by manny-uncharted**
Closes #8512

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by kbdharun — COMMENTED** [review]
@manny-uncharted Can you remove the other 2 files in this PR?

> Seems like you accidentally added it.

File: pages/linux/cvfsck.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# cvfsck
+
+> Check and Recover a Xsan Volume.
+> More information: <https://www.unix.com/man-page/osx/1/cvfsck/>.
+
+- Scan directories for name collisions that would occur on a case-insensitive filesystem:
+
+`cvfsck -A`
+
+- Provide a specific path to a configuration file that is to be used, overriding the implicit location:
```
Comment (kbdharun) [inline pages/linux/cvfsck.md:10]:
```suggestion
- Provide a specific path to a configuration file to be used for overriding the implicit location:
```
I guess think would be better.

File: pages/linux/cvfsck.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+`cvfsck -c {{pathname}}`
+
+- Internal debug use to dump a significant amount of data to the standard output device:
```
Comment (kbdharun) [inline pages/linux/cvfsck.md:14]:
```suggestion
- Dump a significant amount of data to the standard output device (Internal debug use):
```

File: pages/linux/cvfsck.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+> Check and Recover a Xsan Volume.
+> More information: <https://www.unix.com/man-page/osx/1/cvfsck/>.
+
+- Scan directories for name collisions that would occur on a case-insensitive filesystem:
+
+`cvfsck -A`
+
+- Provide a specific path to a configuration file that is to be used, overriding the implicit location:
+
+`cvfsck -c {{pathname}}`
```
Comment (kbdharun) [inline pages/linux/cvfsck.md:12]:
```suggestion
`cvfsck -c {{path/to/configuration_file}}`
```
I think this would be better

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

_Note to maintainers:_
Things I observed in this command.
1. There isn't a manned page for it.
2. It seems like it doesn't support long arguments/flags.

File: pages/linux/cvfsck.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
+# cvfsck
+
+> Check and Recover a Xsan Volume.
+> More information: <https://www.unix.com/man-page/osx/1/cvfsck/>.
```
Comment (navarroaxel) [inline pages/linux/cvfsck.md:4]: This looks like a macOS command, not for Linux.

**Comment by manny-uncharted** [comment]
Yeah, I'll resolve this now

File: pages/osx/cvfsck.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
+# cvfsck
+
+> Check and Recover a Xsan Volume.
+> More information: <https://www.unix.com/man-page/osx/1/cvfsck/>.
```
Comment (navarroaxel) [inline pages/osx/cvfsck.md:4]:
```suggestion
> More information: <https://www.unix.com/man-page/osx/1/cvfsck>.
```

File: pages/osx/cvfsck.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
+# cvfsck
+
+> Check and Recover a Xsan Volume.
```
Comment (navarroaxel) [inline pages/osx/cvfsck.md:3]:
```suggestion
> Check and recover an Xsan volume.
```

File: pages/osx/cvfsck.md [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+> More information: <https://www.unix.com/man-page/osx/1/cvfsck/>.
+
+- Scan directories for name collisions that would occur on a case-insensitive filesystem:
+
+`cvfsck -A`
```
Comment (navarroaxel) [inline pages/osx/cvfsck.md:8]: Does this example also recovers the collisions?

File: pages/osx/cvfsck.md [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+
+- Provide a specific path to a configuration file to be used for overriding the implicit location:
```
Comment (navarroaxel) [inline pages/osx/cvfsck.md:10]: You should explain the whole example, not the `-c` argument.

File: pages/osx/cvfsck.md [hunk 9]
```diff
@@ -0,0 +1,24 @@
...
+
+- Dump a significant amount of data to the standard output device (Internal debug use):
+
+`cvfsck -d`
+
+- Report statistics for extents in each file:
+
+`cvfsck -e`
+
+- Report free space fragmentation:
```
Comment (navarroaxel) [inline pages/osx/cvfsck.md:22]: Where is reported? printed in the terminal?

File: pages/linux/setquota.md [hunk 10]
```diff
@@ -0,0 +1,24 @@
+# setquota
+
+> Set disk quotas for a user or group.
+> More information: <https://manned.org/setquota.8>.
+
+- Edit remote quota over RPC, uses rpc.quotad on the remote host to set quota:
+
+`setquota -r, --remote`
```
Comment (EmilySeville7cfg) [inline pages/linux/setquota.md:8]:
```suggestion
`setquota --remote`
```

- Mandatory fix: [invalid](https://manned.org/setquota.8) command syntax: there is no `-r,` option: man pages can't be often copied without any modifications in TlDr pages.
- Note: long and short options can't be presented now without using placeholder syntax.

File: pages/linux/setquota.md [hunk 11]
```diff
@@ -0,0 +1,24 @@
...
+
+- Set user quotas for named user:
+
+`setquota -u, --user`
```
Comment (EmilySeville7cfg) [inline pages/linux/setquota.md:12]:
```suggestion
`setquota --user`
```

The same issue as above.

File: pages/linux/setquota.md [hunk 12]
```diff
@@ -0,0 +1,24 @@
...
+
+- Set group quotas for named group:
+
+`setquota -g, --group`
```
Comment (EmilySeville7cfg) [inline pages/linux/setquota.md:16]:
```suggestion
`setquota --group`
```

The same issue as above.

File: pages/linux/setquota.md [hunk 13]
```diff
@@ -0,0 +1,24 @@
...
+
+- Use quota settings of user, group or project to set the quota for the named user, group or project:
+
+`setquota -p, --prototype={{protoname}}`
```
Comment (EmilySeville7cfg) [inline pages/linux/setquota.md:20]:
```suggestion
`setquota --prototype={{protoname}}`
```

The same issue as above.

File: pages/linux/setquota.md [hunk 14]
```diff
@@ -0,0 +1,24 @@
...
+
+- Go through all filesystems with quotas in (/etc/mtab) and perform setting:
+
+`setquota -a, --all`
```
Comment (EmilySeville7cfg) [inline pages/linux/setquota.md:24]:
```suggestion
`setquota --all`
```

The same issue as above.

File: pages/osx/setquota.md [hunk 15]
```diff
@@ -0,0 +1,24 @@
+# setquota
```
Comment (EmilySeville7cfg) [inline pages/osx/setquota.md:1]: Mandatory fix: [specification violation](https://github.com/tldr-pages/tldr/blob/main/CLIENT-SPECIFICATION.md#platform): this page can be safely removed: it's the same as `linux/setquota.md`.

> If a page is common across multiple platforms, but slightly differently on a given platform, then the page is still stored in common, but a copy tailored for the differing platform is placed in that platform's specific folder.

**Review by EmilySeville7cfg — CHANGES_REQUESTED** [review]
- Mandatory changes found, changes requested.
- This page should be reviewed by `sequota`, `cvfsck` users.
- Thanks for a contribution! ❤️

**Comment by EmilySeville7cfg** [comment]
According to our [maintainers guide](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/maintainers-guide.md#ii-handling-prs) we can apply suggestions ourselves now. There is no need to wait not responding PR author:

> If the only issues holding up a merge are trivial fixes (typos, syntax errors, etc.), and the author doesn't respond in a day or two, maintainers can make the necessary changes themselves, and proceed with the merge process.

Warning: trivial fixes definition is not complete as it contains `etc` (possible misinterpretation of this term can occur).

**Comment by Waples** [comment]
> According to our [maintainers guide](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/maintainers-guide.md#ii-handling-prs) we can apply mandatory suggestions ourselves now. There is no need to wait not responding PR author:
> 
> > If the only issues holding up a merge are trivial fixes (typos, syntax errors, etc.), and the author doesn't respond in a day or two, maintainers can make the necessary changes themselves, and proceed with the merge process.
> 
> Warning: trivial fixes definition is not complete as it contains `etc` (possible misinterpretation of this term can occur).

@EmilySeville7cfg since this is stuck since December, shall we just apply the suggestions you made then and merge after moving it to /common (if thats even still needed)?

**Review by Waples — CHANGES_REQUESTED** [review]
Please apply the suggestions according to our styleguide (placeholder syntax)

**Comment by EmilySeville7cfg** [comment]
> @EmilySeville7cfg since this is stuck since December, shall we just apply the suggestions you made then and merge after moving it to /common (if thats even still needed)?

It's up to you now, what apply and what not, do what you think is the most appropriate. Since it's usual to make exceptions to style guide I can't decide what is the best, sorry.

**Pull request closed without merging**